psi4/src/psi4/dcft/dcft_compute.cc

```diff
@@ -56,6 +56,9 @@ double DCFTSolver::compute_energy() {
     }
 
     if (options_.get_str("DCFT_TYPE") == "DF") {
+        outfile->Printf(
+            "\n\n\t**** Warning: The density-fitted DCFT cumulant is only approximately variational.\n"
+            "\t     The stationarity conditions, and thus DF-DCFT energy, are subject to change. ****\n");
```

Comment:
stationary, stationality, stationarity?

Reply:
I still prefer "stationarity", but it's not something I'll argue about it if anybody has strong opinions to the contrary.

Reply:
I don't have strong feelings -- just didn't recognize the word. Ok by me so long as intended, not typo.

Reply:
It should IMHO indeed be "stationarity condition", i.e. "condition for a stationary wave function"